webfront/views/modifiers.py

```diff
@@ -902,7 +906,7 @@ def get_model_structure(value, general_handler):
 def get_subfamilies(value, general_handler):
     queryset = general_handler.queryset_manager.get_queryset().first()
     entries = Entry.objects.filter(integrated=queryset.accession, is_alive=False)
-    if value is not None and value.strip() != '':
+    if value is not None and value.strip() != "":
```

Comment:
Empty strings evaluate to `False` so we don't need to compare the value after trimming to an empty string:

```suggestion
    if isinstance(value, str) and value.strip():
```

But if we need to trim `value`, shouldn't the trimmed string be used for filtering entries?

https://github.com/ProteinsWebTeam/interpro7-api/blob/16dba9513f5e53d27ca8f505c77e931d73dfdca8/webfront/views/modifiers.py#L910